src/components/common/RecipientsTable/RecipientTable.js

```diff
@@ -186,140 +158,95 @@ const RecipientTable = ({
       ellipsis: true,
       editable: true,
       render: (_, record) => {
-        const editable = isEditing(record);
-        return editable ? (
-          <Form.Item
-            name="Date of Birth"
-            style={{ margin: 0 }}
-            rules={[
-              {
-                required: true,
-                message: `Please input an date of birth!`,
-              },
-            ]}
-          >
-            <Input value={record.recipient_date_of_birth} />
-          </Form.Item>
-        ) : (
-          <>{record.recipient_date_of_birth}</>
+        return (
+          <div style={{ display: 'flex', justifyContent: 'center' }}>
+            {record.recipient_date_of_birth}
+          </div>
         );
       },
     },
     {
       title: 'Gender',
-      dataIndex: 'gender',
-      key: 'gender',
+      dataIndex: 'gender_id',
+      width: 100,
+      key: 'gender_id',
       filters: [
-        { text: 'Male', value: 'Male' },
-        { text: 'Female', value: 'Female' },
-        { text: 'Non Binary', value: 'Nonbinary' },
+        { text: 'Female', value: 1 },
```

Comment:
Ideally these values would be referenced directly from the back end programmatically instead of being hardcoded. Will need to be fixed in a future PR, please document for next cohort.